packages/cli/test/integration/owner.endpoints.test.ts

```diff
@@ -85,6 +85,23 @@ test('POST /owner should fail with invalid inputs', async () => {
 	}
 });
 
+test('POST /owner/skip-setup should persist skipping setup to the DB', async () => {
+	const owner = await Db.collections.User!.findOneOrFail();
+	const authOwnerAgent = await utils.createAgent(app, { auth: true, user: owner });
+
+	const response = await authOwnerAgent.post('/owner/skip-setup').send();
+
+	expect(response.statusCode).toBe(200);
+
+	const skipConfig = config.get('userManagement.skipInstanceOwnerSetup');
+	expect(skipConfig).toBe(true);
+
+	const { value } = await Db.collections.Settings!.findOneOrFail({
+		key: 'userManagement.skipInstanceOwnerSetup',
+	});
+	expect(value).toBe('true');
```

Comment:
Another setting is using a helper, so a general helper might be best:

```ts
export async function getSetting({ key, type }: { key: string; type: 'boolean' }) {
  const { value } = await Db.collections.Settings!.findOneOrFail({ key });

  if (type === 'boolean') return Boolean(value);

  throw new Error(`Unrecognized setting type: ${type}`);
}
```

Reply:
not everything saved in the settings will be a boolean, so keeping the small call rather to the test than adding another helper